library/zitilib.c

```diff
@@ -587,6 +587,42 @@ static const char* find_service(ztx_wrap_t *wrap, int type, const char *host, ui
     return best;
 }
 
+const char *fmt_identity(const ziti_intercept_cfg_v1 *intercept, const char* proto, const char *host, int port) {
+    if (intercept == NULL) return NULL;
+
+    tag *id_tag = model_map_get(&intercept->dial_options, "identity");
+    if (id_tag == NULL || id_tag->type != tag_string) {
+        return NULL;
+    }
+
+    static char identity[1024];
+    const char *p = id_tag->string_value;
+    char *o = identity;
+    while(*p != 0) {
+        if (*p == '$') {
+            p++;
+            if (strncmp(p, "dst_protocol", strlen("dst_protocol")) == 0) {
```

Comment:
dst_protocol, dst_hostname, dst_port are used here multiple times as well as used in `library/ziti_src.c` and `library/zitilib.c` too... it feel like it should be centralized... maybe defined in a single location?